### PR TITLE
Add Contact and Supporters

### PR DESCRIPTION
**Pull request by JorisGoosen**
Implements https://github.com/jasp-stats/INTERNAL-jasp/issues/2380

- add supporters to welcomepage
- make sure supporters dont collide with new version button
- Add contact window

image

Looks good like this @EJWagenmakers  @JohnnyDoorn  ?

### CODE REVIEW

**Comment by JorisGoosen** [comment]
image

**Comment by JorisGoosen** [comment]
```
For <a href=\"https://github.com/jasp-stats/jasp-issues/issues/new?assignees=&labels=Feature+Request&projects=&template=feature-request.yml&title=%5BFeature+Request%5D%3A+\">feature requests</a> and <a href=\"https://github.com/jasp-stats/jasp-issues/issues/new?assignees=&labels=Bug&projects=&template=bug-report.yml&title=%5BBug%5D%3A+\">bug reports</a>: please post an issue on our GitHub page, <a href=\"https://jasp-stats.org/2018/03/29/request-feature-report-bug-jasp/\">as explained here.</a>
This will bring you in direct contact with the JASP software developers.

For statistical questions: please post an issue <a href=\"https://forum.cogsci.nl/index.php?p=/categories/jasp-bayesfactor\">on the JASP Forum.</a>

For information on the JASP Cooperative: please read <a href=\"https://jasp-stats.org/cooperative/\">the information on the JASP website</a>

For suggesting we add your institution to the <a href=\"https://jasp-stats.org/world-map/\">JASP World Map</a> please send an email to <a href=\"mailto:communications@jasp-stats.org\">communications@jasp-stats.org</a>.

For individual donations: please visit <a href=\"https://jasp-stats.org/donate/\">the JASP website</a>.
```

Is the contact page text

And to add new supporters one just has to add them to the list in `Supporters.qml`

File: Desktop/components/JASP/Widgets/ContactWindow.qml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		textFormat:		Text.RichText
+		text:
+"For <a href=\"https://github.com/jasp-stats/jasp-issues/issues/new?assignees=&labels=Feature+Request&projects=&template=feature-request.yml&title=%5BFeature+Request%5D%3A+\">feature requests</a> and <a href=\"https://github.com/jasp-stats/jasp-issues/issues/new?assignees=&labels=Bug&projects=&template=bug-report.yml&title=%5BBug%5D%3A+\">bug reports</a>: please post an issue on our GitHub page, <a href=\"https://jasp-stats.org/2018/03/29/request-feature-report-bug-jasp/\">as explained here.</a>
+This will bring you in direct contact with the JASP software developers.
+
+For statistical questions: please post an issue <a href=\"https://forum.cogsci.nl/index.php?p=/categories/jasp-bayesfactor\">on the JASP Forum.</a>
+
+For information on the JASP Cooperative: please read <a href=\"https://jasp-stats.org/cooperative/\">the information on the JASP website</a>
+
+For suggesting we add your institution to the <a href=\"https://jasp-stats.org/world-map/\">JASP World Map</a> please send an email to <a href=\"mailto:communications@jasp-stats.org\">communications@jasp-stats.org</a>.
```
Comment (boutinb) [inline Desktop/components/JASP/Widgets/ContactWindow.qml:28]: Does not seem english to me...

File: Desktop/components/JASP/Widgets/Supporters.qml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+					width:					swipeSupporters.width
+				}
+			}
+
+
+		}
+
+		Text
+		{
+			text:					qsTr("Suggest your institution joins the JASP Cooperative").replace(/, /g, ",&nbsp;").replace(/\n\n/g, "<br><br>")
```
Comment (boutinb) [inline Desktop/components/JASP/Widgets/Supporters.qml:82]: Is this sentence from EJ?? My english is not so good, but I don't understand it either... And JASP is not a cooperative.

File: Desktop/components/JASP/Widgets/Supporters.qml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+			id:						swipeSupporters
+			orientation:			Qt.Vertical
+			Layout.fillWidth:		true
+			clip:					true
+			currentIndex:			Math.floor(Math.random() * count)
+			wheelEnabled:			true
+
+			Timer
+			{
+				interval:			5000
```
Comment (boutinb) [inline Desktop/components/JASP/Widgets/Supporters.qml:44]: I would do a bit quicker 4 or even 3 seconds.

**Comment by EJWagenmakers** [comment]
Yes it is English :-)
I like the layout
We are not a cooperation (company) but we are a cooperative, as in different universities cooperating. I am open to a different word

File: Desktop/components/JASP/Widgets/Supporters.qml (hunk 2 above)
Comment (JorisGoosen) [reply Desktop/components/JASP/Widgets/Supporters.qml:82]: Im not making these texts up :p

File: Desktop/components/JASP/Widgets/Supporters.qml (hunk 3 above)
Comment (JorisGoosen) [reply Desktop/components/JASP/Widgets/Supporters.qml:44]: Well, it starts on a different one every time and I already brought it down from 8s

**Comment by JorisGoosen** [comment]
Cooperatives are nice though, although as we aren't a business it is perhaps not applicable.
But consortium sounds so heavy

**Comment by boutinb** [comment]
Good to merge for me. Let's do it, and build, test and release?

**Comment by EJWagenmakers** [comment]
Yes, we have thought about alternatives such as "Coalition". But every term comes loaded with unwanted connotations...

**Comment by boutinb** [comment]
Foundation?

**Comment by EJWagenmakers** [comment]
Yes, but that word is taboo because we have a Foundation but are unsure what to do with it

File: Desktop/components/JASP/Widgets/Supporters.qml (hunk 3 above)
Comment (JohnnyDoorn) [reply Desktop/components/JASP/Widgets/Supporters.qml:44]: yeah you also don't want seizure-inducing changes

File: Desktop/components/JASP/Widgets/Supporters.qml (hunk 2 above)
Comment (JohnnyDoorn) [reply Desktop/components/JASP/Widgets/Supporters.qml:82]: it's a cooperative in the broad sense of the word ^^

**Pull request merged**